autoload/go/doc.vim

```diff
@@ -29,7 +29,19 @@ function! go#doc#OpenBrowser(...) abort
     let name = out["name"]
     let decl = out["decl"]
 
-    let godoc_url = "https://godoc.org/" . import
+    let godoc_url = get(g:, 'go_doc_url', 'https://godoc.org')
+    if godoc_url != 'https://godoc.org'
```

Comment:
Bit of a general issue, but it's better to use `is` and `isnot` instead of `==` and `!=`. The reason for this is:

```
:echo 0 == '0'
1
:echo 0 == '0'
1

:echo 0 is '0'
0
:echo 0 is ''
0
```

In other words, `==` does the automatic type juggling that we like so much in languages such as JavaScript and PHP, and using `is` in Vim is a good idea for the same reason that using `===` in JS/PHP is a good idea.

At this specific comparison it's not really an issue, but I'd like to simply disallow usage of `==` at some point in the future, so it's probably better to get in the habit of using it everywhere.

Reply:
I see your concern. Do you know when `is` or `isnot` was introduced? Is that a new feature? I've didn't saw it much used anywhere else. Also this is just a string comparison, what could wrong here in our case? I'm happy to change it, just want to understand the issue with it.

Reply:
> Do you know when is or isnot was introduced? Is that a new feature?

Somewhere between 7.0 and 7.1.

>  what could wrong here in our case?

Nothing, but I think we should move to a future where `==` is simply forbidden, just like most projects JS and PHP projects forbid using `==` (in favour of `===`).

Reply:
Gotcha. Fixed to `isnot`. Thanks